apps/web/lib/team/[slug]/getServerSideProps.tsx

```diff
@@ -81,6 +88,26 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
     return { notFound: true } as const;
   }
 
+  const organizationSettings = isOrgContext
+    ? team?.isOrganization
+      ? team?.organizationSettings
+      : team?.parent?.organizationSettings
+    : null;
+  let allowSEOIndexing = false;
+  if (!!organizationSettings) {
+    allowSEOIndexing = organizationSettings.allowSEOIndexing ?? false;
+    if (
+      team?.isOrganization &&
+      organizationSettings.orgProfileRedirectsToVerifiedDomain &&
+      organizationSettings.orgAutoAcceptEmail
+    ) {
+      return getRedirectToVerifiedDomain({ verifiedDomain: organizationSettings.orgAutoAcceptEmail });
```

Comment:
```suggestion
      return {
        redirect: {
          permanent: false,
          destination: `https://${verifiedDomain}`,
        },
      };
```

Function just returns the same thing and doesn't do anything additional, we can just keep it simple and return here itself.

Reply:
yeah correct, i had just kept as suggested by hariombalhara, updated